content/blog/2021-09-30-september-21-community-gems.md

```diff
@@ -0,0 +1,127 @@
+---
+title: September '21 Community Gems
+date: 2021-09-30
+description: |
+  A roundup of technical Q&A's from the DVC and CML communities.
+  This month: data registries, working with DVC remotes,
+  queued experiments, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC and CML community.
+  This month: data registries, working with DVC remotes,
+  queued experiments, and more.
+picture: 2021-09-30/gems-cover.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/september-21-community-gems/871
+tags:
+  - Data Registry
+  - DVC Remotes
+  - Queued Experiments
+  - Pipelines
+  - Git
+---
+
+### [If I already have several projects and want to build a data registry out of those?](https://discord.com/channels/485586884165107732/563406153334128681/879651401318400021)
+
+Great question from @m41n314c!
+
+You can import into the registry from your other projects with `dvc import` and
+`dvc add`. Keep in mind that imported data stays stored with the original source
+repo, so people will need to have access to that repo and remote to access the
+imported data in the registry.
+
+Look at the
+[Data Registry use case](https://dvc.org/doc/use-cases/data-registries) where
+you can learn more about data registry management and how other projects can
+interact with the registry.
+
+### [Is there a way to share data across multiple on-premise machines so that users can train models individually?](https://discord.com/channels/485586884165107732/563406153334128681/879718738163826698)
+
+This is a good scenario to try out
+[sharing a DVC cache](https://dvc.org/doc/user-guide/how-to/share-a-dvc-cache).
```

Comment:
we could also mention this use case first - https://dvc.org/doc/user-guide/how-to/share-a-dvc-cache - I think ... btw, @jorgeorpinel it's not clear in the use case that multi-machine scenario is covered also very well

Reply:
@shcheklein I think I'm a little confused on this comment. Should I update the link or update the wording? I did go ahead and update the wording a little bit.

Reply:
I think we should mention two links here - use case is easier to read